### DISCUSSION TITLE
wx.decode is not a function

### DISCUSSION THREAD

**Discussion by Arbureva** [NervJS/taro#13475] [Q & A] [answered]
我需要解析一个文件，小程序有个方法叫 wx.decode。我在taro中没有找到这个方法，直接用又报undefined，请问有小伙伴遇到过这类问题吗？我的需求是在页面 显示的时候取出一个文件的内容，页面隐藏的时候又把新的数据存进去。是个聊天记录储存的功能

**Comment by TheKonka** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/22173084/226779961-52284a98-16d8-4ea7-ae58-4eb159f1d481.png)

这是给小游戏用的，小程序里面用不了
